contracts/Vault.vy

```diff
@@ -219,7 +219,9 @@ debtRatio: public(uint256)  # Debt ratio for the Vault across all strategies (in
 totalDebt: public(uint256)  # Amount of tokens that all strategies have borrowed
 lastReport: public(uint256)  # block.timestamp of last report
 activation: public(uint256)  # block.timestamp of contract deployment
+lockedProfit: public(uint256) # how much profit is locked and cant be withdrawn
```

Comment:
If we add a mapping of per-strategy profit locked (or add it to the struct), we could make this an aggregated value, and the adjust the aggregated value during report (subtract the old per-strategy value, then add the new)

Reply:
i think that ideas is good since with more strategies overwriting last Report could become an issue. Main thing is we can make sure it doesnt over complicate the change, since this is an improvement already over current code.

Reply:
Turn this into an issue, and we'll loop back to it later

Reply:
Done
https://github.com/iearn-finance/yearn-vaults/issues/196